webpages/popup/index.js

```diff
@@ -53,15 +59,18 @@ const vue = new Vue({
   },
   computed: {
     changelogLink() {
-      const uiLanguage = chrome.i18n.getUILanguage();
-      const localeSlash = uiLanguage.startsWith("en") ? "" : `${uiLanguage.split("-")[0]}/`;
-      const utm = `utm_source=extension&utm_medium=popup&utm_campaign=v${chrome.runtime.getManifest().version}`;
-      return `https://scratchaddons.com/${localeSlash}changelog/?${utm}#v${chrome.runtime.getManifest().version}`;
+      if (this.devMode) {
+        return `https://github.com/ScratchAddons/ScratchAddons/commits/${commitHash}`;
+      } else {
+        const uiLanguage = chrome.i18n.getUILanguage();
+        const localeSlash = uiLanguage.startsWith("en") ? "" : `${uiLanguage.split("-")[0]}/`;
+        const utm = `utm_source=extension&utm_medium=popup&utm_campaign=v${chrome.runtime.getManifest().version}`;
+        return `https://scratchaddons.com/${localeSlash}changelog/?${utm}#v${chrome.runtime.getManifest().version}`;
+      }
     },
     version() {
-      const prerelease = chrome.runtime.getManifest().version_name.includes("-prerelease");
       const ver = chrome.runtime.getManifest().version;
-      return prerelease ? ver + "-pre" : ver;
+      return this.devMode ? ver + "-pre" : ver;
     },
```

Comment:
Not sure if this should base off of `this.devMode`. What does this function do?

Reply:
It displays `-pre` after the version number in the popup, so yeah it might make sense to revert that.